docs/api-docs/getting-started/webhooks/webhook-events.md

```diff
@@ -443,6 +445,32 @@ Updates to the following fields trigger a `store/channel/updated` event.
 }
 ```
 
+### The same response is returned for the following events:
+
+* `store/order/transaction/created`
+* `store/order/transaction/updated`
+
+**Response fields**
+- type -- Will always be transaction
+- order_id -- ID of the order associated with the transaction
+- transaction_id -- ID of the transaction created by the payment processor

```

Comment:
`- transaction_id -- ID of the transaction created by the payment provider`

Reply:
Thanks for the feedback. That change has been made.